packages/legacy-scripting-runner/test/bundle.js

```diff
@@ -52,6 +53,7 @@ describe('bundleConverter', () => {
       standard_poll: true,
       page: 1,
       limit: 100,
+      isBulkRead: false,
```

Comment:
if there's a default in the code itself, you may not need all these additions

Reply:
I was hoping that would be the case, but without these additions, the tests failed. I only added this in places where the tests were failing -- if there's a better way to do it I'd definitely be for that instead!

Reply:
ok, no worries! Not sure what's up there, but it's probably fine.